core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala

```diff
@@ -111,17 +114,40 @@ object TiUtil {
                               blacklist: ExpressionBlacklist): Boolean =
     isSupportedBasicExpression(expr, source, blacklist) && isPushDownSupported(expr, source)
 
+  def toSparkRow(row: TiRow, rowTransformer: RowTransformer, version: Int): Row = {
```

Comment:
If you need pass version around, I'd rather make it an class with version as field.